workflow/CustomMoment/CustomMoment.js

```diff
@@ -0,0 +1,150 @@
+(function () {
+    class CustomMoment {
+        constructor(date) {
+            if (typeof date == "number") {
+                this.date = new Date(date)
+            }
+            else this.date = date
+        }
+
+        static parse(dateToParse, formatToParse) {
+            let result;
+            let days = ""
+            let months = ""
+            let years = ""
+            let newArray = []
+            let splitArrayDate = ""
+            switch (formatToParse) {
```

Comment:
what if i want only YYYY or only DD ?
Please, do not use switch/case here, you have to parse date your format string by parts and then get this parts from the date string.